charts/substra-tests-stack/templates/rbac.yaml

```diff
@@ -40,5 +28,17 @@ subjects:
   namespace: {{ .Release.Namespace }}
 roleRef:
   kind: ClusterRole
-  name: cluster-admin # required by `helm init` in `deploy.sh`
+  name: tiller-user
   apiGroup: rbac.authorization.k8s.io
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tiller-user
+  namespace: kube-system
+rules:
+rules:
+- apiGroups: ["", "batch", "extensions", "apps"]
+  resources: ["*"]
+  verbs: ["*"]
```

Comment:
Helm really needs these access to run ? In the docs they just reference `pods/portforward` and `pods`.
This is the role your pod runnning helm will use, not the role for the tiller pod.
Your tiller pod should already be created and cluster-admin according to this line : https://github.com/SubstraFoundation/substra-tests/blob/553994e918e760568068c313b8d478b803e72be2/ci/run-ci.sh#L138 .

Reply:
I've tested with the documentation access, but it does not work. I have to spend more time to see the real apiGroups, resources and verbs needed.

Reply:
It seems that the documentation show miniaml rbac for helm to interact with tiller but only interact ^^

Reply:
But helm should only interact with tiller as tiller is the one in charge of creating ressources in the k8s cluster.
I think the issue is related to the fact that we have the --upgrade flag on the helm init command here: https://github.com/SubstraFoundation/substra-tests/blob/143cc4a76b80e4fad9d036808bfebf120f66cf1f/docker/substra-tests-stack/bin/deploy.sh#L10 So helm needs the rights to deploy a new tiller.

But I think the upgrade is mandatory as you can't be certain of the helm version the user used to run the ci/run-ci.sh script.

Reply:
Moreover, we heavily use skaffold which need to operate on pods, etc. for the build and the run part.

Reply:
@AlexandrePicosson  I've spend some time on it and updated https://github.com/SubstraFoundation/substra-tests/pull/88/commits/f7282e8b9903f782dc76435aafdc9a38211107fc

If you have some ideas feel free to contribute with a commit :) !